code/modules/mob/dead/observer/orbit.dm

```diff
@@ -58,12 +58,14 @@
 
 		var/mob/M = poi
 		if(istype(M))
+			if(isnewplayer(M))  // People in the lobby screen; only have their ckey as a name.
+				continue
 			if(isobserver(M))
 				ghosts += list(serialized)
-			else if(M.stat == DEAD)
-				dead += list(serialized)
 			else if(M.mind == null)
 				npcs += list(serialized)
+			else if(M.stat == DEAD)
+				dead += list(serialized)
```

Comment:
What's the significance of moving this check down one? Is it because we're more likely to encounter a mindless mob (NPC) over a dead one?

Reply:
Not that we are "more likely", but that when we DO encounter a dead mindless mob, it hits the first branch (mindless) and gets counted as an NPC, rather than hitting the second one (dead) and gets grouped with dead players.

Reply:
Gotcha.